cobalt/media/base/metrics_provider.cc

```diff
@@ -151,6 +151,27 @@ void MediaMetricsProvider::ReportActionLatencyUMA(
       UMA_HISTOGRAM_TIMES("Cobalt.Media.SbPlayer.Destroy.LatencyTiming",
                           action_duration);
       break;
+    case MediaAction::SBDRM_DESTROY_SYSTEM:
+      UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
+          "Cobalt.Media.SbDrm.Destroy.LatencyTiming", action_duration,
+          base::TimeDelta::FromMicroseconds(500),
+          base::TimeDelta::FromMilliseconds(250), 50);
+      break;
+    case MediaAction::SBDRM_GENERATE_SESSION_UPDATE_REQUEST:
+      UMA_HISTOGRAM_TIMES(
```

Comment:
This histogram ranges from 1ms to 10s. The upper bound in particular is way higher than your estimate. FYI. You ok with this?  Same with your other histograms below.

Reply:
Yes, as the average is about ~173ms for `SbDrmGenerateSessionUpdateRequest()`, it is expected that the histogram would look like `SbPlayerDestroy()` in https://github.com/youtube/cobalt/pull/3822.

As it is possible that there are many other slower devices, keeping it in the range of 1ms to 10s can ensure we can capture the distribution.